src/en/proposals/errant-vox-species.md

```diff
@@ -0,0 +1,94 @@
+
+#### Difficult
+They are not intended to be perfectly balanced, or even necessarily what you might call "on par". Their main "feature" is in itself very punishing. If they overall fall towards the weaker side of the meta, that's just the cost of being vox. While they can and should have some positives, they should not be given big buffs just to "balance out" their shortcomings. They should not be prohibitively difficult, but if they are a less popular choice, that fits just fine into Difficult.
+
+#### Cooperative
+Historically, due to the difficulty of surviving a round as a vox, they had a tendency to rely on each other to get by in a world that was just not built for them. When no one else "got it", vox players turned for support to those who did. (Or so I'm told). It is the position of this design that this cooperative aspect of them is something to be preserved, and it being species specific is not something that should either be encouraged or discouraged mechanically, leaving it to the players to decide.
+Obviously, "just designating" how a species is supposed to play like is an impossible, lofty goal, but nevertheless their features could be designed in a way to help with that
+This was the hardest pillar to phrase and will be the hardest to come up with ideas for. But if a feature works better with other players, does something for other players rather than you, or is a flaw that is more easily counteracted with external help rather than just having the right tool in your pocket, then that feature is Cooperative.
+
+
+## Specific Features and Ideas 
+(All subject to further review during the actual implementation, of course. Especially towards the end of the list. They are mostly just intended as a showcase of ideas to start out with)
+
+  - There should be some sort of warning or confirmation when selecting them, so players, especially new players, don't end up unwittingly picking a choice that will kill their character for the wrong click. 
+
+  - They breathe nitrogen, but oxygen hurts them badly (tentative number is, they crit in 2 minutes in normal air. Afaik this is actually more lenient than in Spess13)  - they will start with a large nitrogen tank and breathing mask equipped
+    - because taking their mask off is BAD, eating and drinking is challenging. They need a nitrogen room, and someone needs to have made that room
+    - possible alternative to a nitrogen room is the as-yet uncoded feature to hold their breath, taking asphyxiation damage instead of poison (at an even faster rate). They can regenerate that automatically (assuming they don't miscalculate and go crit)
```

Comment:
I prefer this more to requiring mappers to add/make sure it's possible to have a nitrogen room. Holding your breath is something I'm very likely going to add to newMed. Holding your breath gives the player more agency when trying to solve their breathing problem rather than just simply dying if there is not nitrogen room nearby.

Reply:
(Another alternative is to just give them a mask that does not block eating, but that's terribly lame so I'm only bringing up the idea to shoot it down)

Reply:
Yeah I have decided that hold breath is something that they will have to be able to do by release. 
But for the record, if they hold their breath, that will crit them FASTER than just breathing the toxins would have had. (this is not me being mean, it's how our systems already work. Asphyxiation crits in about 40 seconds.) 
The advantage there is that you can **recover** that damage afterwards just by waiting, unlike the poison damage you would take from breathing the oxygen